src/plugins/workflow_designer/src/library/ConvertFilesFormatWorker.cpp

```diff
@@ -187,7 +186,14 @@ void ConvertFilesFormatWorker::init() {
     inputUrlPort = ports.value(INPUT_PORT);
     outputUrlPort = ports.value(OUTPUT_PORT);
     targetFormat = getValue<QString>(BaseAttributes::DOCUMENT_FORMAT_ATTRIBUTE().getId());
-    excludedFormats = getValue<QString>(EXCLUDED_FORMATS_ID).split(",", QString::SkipEmptyParts);
+    QStringList excludedFormatNames = getValue<QString>(EXCLUDED_FORMATS_ID).split(",", QString::SkipEmptyParts);
```

Comment:
BaseAttributes::DOCUMENT_FORMAT_ATTRIBUTE  has type and the type is "DOCUMENT_FORMAT_ATTR_ID"
It can't be name it must always be id.

The correct fix is to change the rendering of the format id in combo-box-> use "name" as a rendered value, but do not change what is passed inside WD bus during workflow run.

The WD bus content may be passed not only from the combo-box you tweaked, but also from the command line params, WD-schema file, etc.